### ISSUE TITLE
Add doc-aliases to functions that have their name changed from godot

### ISSUE THREAD

**Issue by sayaks** [godot-rust/gdext#232]
Several functions have had their function-names changed from godot's name, usually to better match idiomatic rust naming conventions. Such as renaming various `get_x()` functions to just `x()`. In addition, we sometimes remove functions in favor of other functions that have identical functionality.

To aid users with finding these functions they should have a `#[doc(alias)]` referencing the original function name/the function that was removed in favor of it.

**Comment by Bromeon** [comment]
Yep, I was recommending this in individual reviews, but it's great to have a focused effort, and document it in the guidelines! 👍

**Comment by Bromeon** [comment]
How do we want to approach this concretely? This is somewhat open-ended, unless someone volunteers to invest the effort to go through all APIs and check all the renames... 

Otherwise I'd like to have issues that have a clear actionable, so that they can be closed at some point and don't clutter the tracker for eternity. This could also be done as an incremental effort, I think 🙂
